### DISCUSSION TITLE
Number inputs being rounded....

### DISCUSSION THREAD

**Discussion by MLazich-MVP** [microsoft/AdaptiveCards#8869] [Q&A]
Prior to 4/1 my adaptive card accepted decimal numbers as an input. Now it seems to round decimal numbers to whole numbers.. was this a planned change? Any workarounds outside of entering as a string, obviously entering in a dollar amount and having it round is not ideal. 

https://github.com/microsoft/AdaptiveCards/assets/73117300/d7139f87-4c3c-48d1-8aca-b92e379fae92

**Comment by jwoo-msft** [comment]
I don't believe we have made such drastic changes recently. Could this be due to changes made by Apps that are using AdpativeCards library?

**Reply by MLazich-MVP** [reply]
this is an adaptive card within Teams. Here is an example of one of the items.

`"items": [
                                {
                                    "type": "Input.Number",
                                    "placeholder": "Enter Value",
                                    "isRequired": true,
                                    "id": "a.kpiMonthlyTotalforCSP",
                                    "min": 0.00,
                                    "value": 0.00,
                                    "errorMessage": "Please enter the current Monthly Total for the CSP Service Line (0.00)"
                                }
                            ]`

**Reply by jwoo-msft** [reply]
@MLazich-FSI ,

You can verify your payload using our Adaptive Card designer to see if the behavior remains consistent. If the issue persists, it should be addressed to the Teams team, as it appears to be originating from there.
https://adaptivecards.io/designer/

**Reply by MLazich-MVP** [reply]
sounds good ill see if I can find the appropriate channel for them.
